src/ucs/sys/topo/base/topo.c

```diff
@@ -412,8 +411,7 @@ static void ucs_topo_get_memory_distance_sysfs(ucs_sys_device_t device,
                                             ucs_numa_node_of_cpu(cpu));
     }
 
-    dev_name            = ucs_topo_sys_device_get_name(device);
-    distance->bandwidth = ucs_topo_get_pci_bw(dev_name, path);
+    distance->bandwidth = ucs_topo_default_distance.bandwidth;
```

Comment:
What is the meaning of this line?
Does it mean that "sysfs" topology provider has no impact on BW?

Reply:
We define distance as a pair, latency and BW.
In theory this function can return only latency because memory NUMA distance affects only the latency, but I think that using only one definition for distance is more complete.

Max BW act as the "identity element" for BW.

Reply:
If the function returns different latency values and always returns MAX BW, shouldn't we use smth like: 
```
typedef double (*ucs_topo_get_memory_latency_func_t)(ucs_sys_device_t device);
```
instead of:
```
typedef void (*ucs_topo_get_memory_distance_func_t)(
        ucs_sys_device_t device, ucs_sys_dev_distance_t *distance);
```
?